packages/boot/src/__tests__/acceptance/crud-rest.api-builder.acceptance.ts

```diff
@@ -165,7 +165,6 @@ module.exports = {
   }
 
   async function stopApp() {
-    if (app.state !== 'started') return;
-    await app.stop();
+    if (app?.state === 'started') await app?.stop();
```

Comment:
Can you please preserve the reverse logic? I find such code easier to understand. I am also not sure how easy it is to set a breakpoint on `app?.stop` when it's on the same line as the `if` statement.

```ts
if (app?.state !== 'started') return;
await app?.stop();
```

The same comment applies to `packages/boot/src/__tests__/acceptance/model-api.booter.acceptance.ts` below too.

Reply:
It's easy to set up a breakpoint on `if (app?.state === 'started') await app?.stop();` and step into `app.stop()`.